src/Adapter/LegacyContext.php

```diff
@@ -333,4 +343,27 @@ public function getAvailableLanguages()
     {
         return $this->getLanguages(false);
     }
+
+    /**
+     * This will transfer registered CSS and JS assets
+     * from previous context controller instance
+     * to AdminLegacyLayoutController
+     *
+     * @param Controller $previousContextController
+     * @param AdminLegacyLayoutControllerCore $originCtrl
+     */
+    private function preserveCssAndJSAssets($previousContextController, AdminLegacyLayoutControllerCore $originCtrl): void
```

Comment:
```suggestion
    private function transferCssAndJSAssets($previousContextController, AdminLegacyLayoutControllerCore $originCtrl): void
```

or `copyCssAndJSAssets` would be easier to understand

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19277/commits/db3bcbb7490699d6071d2094d1ab1f708bb3220c